pkg/integration/components/views.go

```diff
@@ -36,98 +39,129 @@ func (self *Views) Secondary() *ViewDriver {
 	}
 }
 
-func (self *Views) byName(viewName string) *ViewDriver {
+func (self *Views) regularView(viewName string) *ViewDriver {
```

Comment:
not really happy with the name here but needed to distinguish from a view driver that had its own special getSelectedLines function